features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageTypeResolver.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.base64
+
+import android.graphics.drawable.Drawable
+import android.graphics.drawable.GradientDrawable
+import androidx.annotation.VisibleForTesting
+
+internal class ImageTypeResolver {
+    fun isDrawablePII(drawable: Drawable, widthInDp: Int, heightInDp: Int): Boolean =
+        drawable !is GradientDrawable &&
+            (
+                widthInDp >= IMAGE_DIMEN_CONSIDERED_PII_IN_DP ||
```

Comment:
here you are using DP but I see that you are calling it with a `densityNormalized` value.

Reply:
`DensityNormalized` divides the input by the density. Does this not mean dp? (pixels/density = dp)